Flaky-test: PersistentTopicTest.testAccumulativeStats is flaky

<!--- 

Instructions for reporting a flaky test using this issue template:

1. Replace [test class] in title and body with the test class name
2. Replace [test method] in title and body with the test method that failed. Multiple methods are flaky, remove the content that refers to the test method.
3. Replace "url here" with a url to an example failure. In the Github Actions workflow run logs, you can right click on the line number to copy a link to the line. Example of such url is https://github.com/apache/pulsar/pull/8892/checks?check_run_id=1531075794#step:9:377 . The logs are available for a limited amount of time (usually for a few weeks).
4. Replace "relevant parts of the exception stacktrace here" with the a few lines of the stack trace that shows at leat the exception message and the line of test code where the stacktrace occured.
5. Replace "full exception stacktrace here" with the full exception stacktrace from logs. This section will be hidded by default.
6. Remove all unused fields / content to unclutter the reported issue. Remove this comment too.

-->
org.apache.pulsar.broker.service.persistent.PersistentTopicTest is flaky. The testAccumulativeStats test method fails sporadically.

[example failure](https://github.com/apache/pulsar/pull/10807/checks?check_run_id=2735643757#step:10:297)

```
Error:  Tests run: 13, Failures: 4, Errors: 0, Skipped: 9, Time elapsed: 3.536 s <<< FAILURE! - in org.apache.pulsar.broker.service.persistent.PersistentTopicTest
Error:  testAccumulativeStats(org.apache.pulsar.broker.service.persistent.PersistentTopicTest)  Time elapsed: 0.104 s  <<< FAILURE!
java.lang.AssertionError: expected [118] but found [177]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:797)
	at org.testng.Assert.assertEquals(Assert.java:807)
	at org.apache.pulsar.broker.service.persistent.PersistentTopicTest.testAccumulativeStats(PersistentTopicTest.java:240)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

--- a/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentTopicTest.java
+++ b/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentTopicTest.java
@@ -70,7 +70,6 @@ public class NonPersistentTopicTest extends BrokerTestBase {
         assertEquals(stats.msgOutCounter, 0);
 
         producer.newMessage().value("test").eventTime(5).send();
-        producer.newMessage().value("test").eventTime(5).send();
 
         Message<String> msg = consumer1.receive();
         assertNotNull(msg);

--- a/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentTopicTest.java
+++ b/pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentTopicTest.java
@@ -213,7 +213,6 @@ public class PersistentTopicTest extends BrokerTestBase {
         assertEquals(stats.msgOutCounter, 0);
 
         producer.newMessage().value("test").eventTime(5).send();
-        producer.newMessage().value("test").eventTime(5).send();
 
         Message<String> msg = consumer1.receive();
         assertNotNull(msg);

